app/src/main/java/com/codepath/michfeng/songswiper/fragments/SwipeFragment.java

```diff
@@ -48,13 +63,19 @@ public static SwipeFragment newInstance(String param1, String param2) {
         return fragment;
     }
 
+
     @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        if (getArguments() != null) {
-            mParam1 = getArguments().getString(ARG_PARAM1);
-            mParam2 = getArguments().getString(ARG_PARAM2);
-        }
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        tv = (TextView) view.findViewById(R.id.textView);
+
+        recommendationService = new RecommendationService(getApplicationContext());
+        recommendationService.get(() -> {
+            this.recommendations = recommendationService.getRecommendations();
+        });
+
+        // At this point, recommendations is still null. Possibly an issue with scope.
+        Log.i("SwipeFragment","recommended tracks: "+recommendations.toString());
```

Comment:
For this Github repo, there may not be a linter set-up, but generally the linter will want spaces after every comma, as well as spaces between operators, for ex:
```
Log.i("SwipeFragment", "recommended tracks: " + recommendations.toString());
```